build.gradle

```diff
@@ -1080,6 +1080,17 @@ project('spring-integration-bom') {
     }
 }
 
+
+project('spring-integration-hazelcast') {
+    description = 'Spring Integration Hazelcast Support'
+    dependencies {
+	implementation "com.hazelcast:hazelcast:$hazelcastVersion"
+	api project(':spring-integration-core')
+
+	testImplementation project(':spring-integration-test')
```

Comment:
I'm not sure why we need this.
Perhaps existing, imported to every modules `testImplementation(project(':spring-integration-test-support')) {` is enough...